### ISSUE TITLE
Question: fine-tuning transformers always requires data to be in the form [sentence1, sentence2, label]?

### ISSUE THREAD

**Issue by andreamorgar** [UKPLab/sentence-transformers#1714]
Can I train a sentence-transformer in the form [sentence1, label]? 
Would it be correct?  Can't find neither in the documentation or hugging face nothing related to this specific way of fine-tuning SBERT. It always relate to two sentences per training item!

**Comment by sarthak91-debug** [comment]
I had the same doubt, I want to experiment with models using SBERT and pass only a text through an encoder with fewer lines of code.

**Comment by TheTamT3** [comment]
@andreamorgar No, You can choose another ways. https://www.sbert.net/docs/package_reference/losses.html#mseloss
